spec/system/managing_topics_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics' do
+  before do
+    FactoryBot.create(:topic, name: 'What is your favourite colour')
+  end
+
+  scenario 'when viewing' do
+    visit('/topics')
+    expect(page).to have_content('Topics')
+    expect(page).to have_content('What is your favourite colour')
+  end
+
+  scenario 'can navigate to new topic page' do
+    visit('/topics')
+    expect(page).to have_link('Add your own topics here',href: new_topic_path)
+  end
+
+  scenario 'when creating a topic' do
+    visit '/topics/new'
+    fill_in 'Name', with: 'tell us a story'
+    click_button 'Create'
+    expect(page).to have_content 'tell us a story'
```

Comment:
This isn't a good test that the topic was actually created. This expectation will pass if, for example, the topic failed validation for some reason and the new page was represented.

Instead, add an expectation that confirms you are on the expected page (in this case the :index page), before checking that the topic is present.